packages/core/config/storybook/main.js

```diff
@@ -29,6 +29,10 @@ const baseConfig = {
       }
     }
 
+    if (fs.existsSync(getPaths().web.storybookPreviewConfig)) {
+      sbConfig.resolve.alias['~__REDWOOD__USER_STORYBOOK_PREVIEW_CONFIG'] = getPaths().web.storybookPreviewConfig
```

Comment:
since `getPaths()` needs to run in a node context, we can't run it from `preview.js` directly. Instead, I elected to run it on this side and expose the path to the user's preview file as a Webpack alias.

Reply:
That's 100% the way. I meant to document that in the comments, I hope it wasn't too painful to figure out.